serverless-logic/serverless-logic-managing-services/serverless-logic-configuring-openAPI-services-endpoints.adoc

```diff
@@ -18,7 +18,7 @@ The `kogito.sw.operationIdStrategy` property supports the following values: `FIL
 quarkus.rest-client.stock_portfolio_svc_yaml.url=http://localhost:8282/ # <1>
 ----
 +
-<1> The OpenAPI File Path is `src/main/resources/openapi/stock-portfolio-svc.yaml`. The generated key that configures the URL for the REST client is `stock_portfolio_svc_yaml`
+<1> The OpenAPI File Path is `<project_application_dir>/specs/stock-portfolio-svc.yaml`. The generated key that configures the URL for the REST client is `stock_portfolio_svc_yaml`
```

Comment:
This is probably not the scope of this PR but maybe "The OpenAPI File Path" should be "The OpenAPI file path" (lowercase) OR "`FILE_PATH`" ?

Reply:
I considered this complete. Feel free to send a new PR to address this issue or report an issue to https://issues.redhat.com/projects/SRVLOGIC/issues if you think this needs to be followed up.